ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -592,84 +86,14 @@
 			buildActionMask = 2147483647;
 			files = (
 				ED297163215061F000B7C4FE /* JavaScriptCore.framework in Frameworks */,
-				3C289EE01FF361C9002AF37A /* libRCTPushNotification.a in Frameworks */,
-				A146BE491FDB4C640090EA06 /* libART.a in Frameworks */,
-				3C4249EB1EF6E09F00D245F1 /* libRNNotifications.a in Frameworks */,
-				CFCFE5491F00158500C295CF /* libRCTCameraRoll.a in Frameworks */,
-				A14EA8771EACE522009D9E83 /* libRCTAnimation.a in Frameworks */,
-				146834051AC3E58100842450 /* libReact.a in Frameworks */,
-				00C302E51ABCBA2D00DB3ED1 /* libRCTActionSheet.a in Frameworks */,
-				00C302E71ABCBA2D00DB3ED1 /* libRCTGeolocation.a in Frameworks */,
-				00C302E81ABCBA2D00DB3ED1 /* libRCTImage.a in Frameworks */,
-				133E29F31AD74F7200F7D852 /* libRCTLinking.a in Frameworks */,
-				00C302E91ABCBA2D00DB3ED1 /* libRCTNetwork.a in Frameworks */,
-				139105C61AF99C1200B5F7CC /* libRCTSettings.a in Frameworks */,
-				832341BD1AAA6AB300B99B32 /* libRCTText.a in Frameworks */,
-				00C302EA1ABCBA2D00DB3ED1 /* libRCTVibration.a in Frameworks */,
-				139FDEF61B0652A700C62182 /* libRCTWebSocket.a in Frameworks */,
-				05C34EA2C79448BB9767887A /* libRNVectorIcons.a in Frameworks */,
-				D6D40984ED07473499B3F0C9 /* libRNDeviceInfo.a in Frameworks */,
-				78FA95D8058C4372AB199525 /* libRNSound.a in Frameworks */,
-				A83E9861943143E6A10A689E /* libSafariViewManager.a in Frameworks */,
-				ADE3B79538D54689B14FEB10 /* libRNFetchBlob.a in Frameworks */,
-				2318BF63A79049CA9F97E2E0 /* libRNSentry.a in Frameworks */,
 				840D44FCCBB14F97B77D9443 /* libz.tbd in Frameworks */,
```

Comment:
Also puzzled about this one.

This one isn't in the upstream template -- but it also looks like it never was. It entered our project file in a18c95b59 (#881).

Well, I guess that makes any cleanup of it orthogonal to this PR, then.

Reply:
> Well, I guess that makes any cleanup of it orthogonal to this PR, then.

OK, noted.